### ISSUE TITLE
ABT wrongly forces prompt when leaving post edition!

### ISSUE THREAD

**Issue by necxelos** [dsifford/academic-bloggers-toolkit#376]
For some strange reason whenever I edit a post with citations in it and save it (update) afterwards, leaving the post edit page still shows me the prompt "your data might not have been saved, please confirm that you want to leave". This happens even if I just enter the edit page and try to leave it without having done anything at all.

It's only the case with ABT though, as posts with no citations added (plugin still active though) doesn't have this issue. It also prolongs the time of loading edit page of posts with citations and time of exiting edit page of posts with citations which makes me believe there might be some bottleneck going on.

**Comment by dsifford** [comment]
No bottleneck. This is a necessary step for the browser because even though you do not make any changes to the editor yourself, each time the edit page is loaded the plugin has to run through the editor and capture all citations to rebuild the state (even if none exist). 

During the process of this, the editor is "dirtied" because a bookmark is set and and removed from it during this process. Because the editor content was modified and reverted (even for a few milliseconds), the browser takes note of this and displays the warning. 

TL;DR - No way to avoid it. Not a performance bottleneck.
